Enhancement/CIAC-16873/ES add mirror (#45164)
demisto/content: Packs/Elasticsearch/Integrations/Elasticsearch_v2/README.md

| Incident type |  | False |
| Fetch incidents |  | False |
| Space ID | The default Kibana space ID to use for es-kibana-* commands. Used to derive the Kibana base URL from the Server URL. If a `space_id` argument is provided to a command, it overrides this value. See [Spaces](https://www.elastic.co/docs/deploy-manage/manage-spaces) for more information. | False |
| Fetch incident types | The type of incident to fetch from Elasticsearch (Elasticsearch Entity, Elasticsearch Security Alert, or Elasticsearch Case). | False |

> | Incident Mirroring Direction | Mirroring works only for incident types Elasticsearch Security Alert and Elasticsearch Case. | False |

Reviewer: ```suggestion
| Incident Mirroring Direction | Mirroring works only for the Elasticsearch Security Alert and Elasticsearch Case incident types. | False |
```

Reviewer: I'm confused by this. It seems like its stating a fact but its a true/false and im not sure what the setting is for. 

Should it be "Whether mirroring works only for the Elasticsearch Security Alert and Elasticsearch Case incident types." 

Or should it be something like "Whether mirroring works in both directions. Mirroring works only for the Elasticsearch Security Alert and Elasticsearch Case incident types."

> | Fetch cases by Severity | Case severities to fetch. If none is chosen, all severity levels will be returned. Relevant only for incident type Elasticsearch Case. | False |

Reviewer: ```suggestion
| Fetch cases by Severity | Case severities to fetch. If none are chosen, all severity levels will be returned. Relevant only for incident type Elasticsearch Case. | False |
```

> | Fetch cases by Status | Case statuses to fetch. If none is chosen, in-progress and open will be returned. Relevant only for incident type Elasticsearch Case. | False |

Reviewer: ```suggestion
| Fetch cases by Status | Case statuses to fetch. If none are chosen, in-progress and open will be returned. Relevant only for incident type Elasticsearch Case. | False |
```

> | Fetch alerts for case | When enabled, fetches alerts associated with each Elasticsearch Case. You must have read privileges for the Cases feature in the Management, Observability, or Security section of the Kibana feature privileges. | False |

Reviewer: ```suggestion
| Fetch alerts for case | Whether to fetch alerts associated with each Elasticsearch Case. You must have read privileges for the Cases feature in the Management, Observability, or Security section of the Kibana feature privileges. | False |
```

> | Close Mirrored XSOAR Incident | When selected, closing the Elasticsearch alert or case is mirrored in Cortex XSOAR. | False |

Reviewer: ```suggestion
| Close Mirrored XSOAR Incident | Whether closing the Elasticsearch alert or case is mirrored in Cortex XSOAR. | False |
```

> | Close Mirrored Elasticsearch Incident | When selected, closing the Cortex XSOAR incident is mirrored in Elasticsearch. | False |

Reviewer: ```suggestion
| Close Mirrored Elasticsearch Incident | Whether closing the Cortex XSOAR incident is mirrored in Elasticsearch. | False |
```
